<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartGameDialog</class>
 <widget class="QDialog" name="StartGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>対局</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>先手／下手</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>601</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="radioButtonHuman1">
         <property name="text">
          <string>人間</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelPlayer1">
         <property name="text">
          <string>名前</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditHumanName1">
         <property name="text">
          <string>You</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QRadioButton" name="radioButtonEngine1">
         <property name="text">
          <string>エンジン</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxEngine1"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonEngineSettings1">
         <property name="text">
          <string>エンジン設定</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>621</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>後手／上手</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>601</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QRadioButton" name="radioButtonHuman2">
         <property name="text">
          <string>人間</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelPlayer2">
         <property name="text">
          <string>名前</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditHumanName2">
         <property name="text">
          <string>You</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QRadioButton" name="radioButtonEngine2">
         <property name="text">
          <string>エンジン</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxEngine2"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonEngineSettings2">
         <property name="text">
          <string>エンジン設定</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxSecondPlayerTimeSettings">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>180</y>
     <width>621</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>後手／上手に異なる時間を設定</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>持ち時間</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="basicTimeHour2"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>時間</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="basicTimeMinutes2">
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>分</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>秒読み    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="byoyomiSec2">
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>秒</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>増加        </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="addEachMoveSec2">
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>秒</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>590</y>
     <width>221</width>
     <height>39</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>621</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>時間設定</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>持ち時間</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="basicTimeHour1"/>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>時間</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="basicTimeMinutes1">
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>分</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>秒読み    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="byoyomiSec1">
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>秒</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_11">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>※ 秒読みと増加は併用できません</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_14">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>増加        </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="addEachMoveSec1">
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>秒</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_12">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonSwapSides">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>590</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>↔　先後入れ替え</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>1251</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>設定</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>開始局面</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxStartingPosition">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>現在の局面</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>平手</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>香落ち</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>右香落ち</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>角落ち</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>飛車落ち</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>飛香落ち</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>二枚落ち</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>三枚落ち</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>四枚落ち</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>五枚落ち</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>左五枚落ち</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>六枚落ち</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>八枚落ち</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>十枚落ち</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>最大手数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxMaxMoves">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>連続対局</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxConsecutiveGames">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_17">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QCheckBox" name="checkBoxShowHumanInFront">
          <property name="text">
           <string>人を手前に表示する</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_16">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QCheckBox" name="checkBoxAutoSaveKifu">
          <property name="text">
           <string>棋譜自動保存</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_15">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBoxLoseOnTimeOut">
          <property name="text">
           <string>時間切れを負けにする</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_13">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <widget class="QCheckBox" name="checkBoxSwitchTurnEachGame">
          <property name="text">
           <string>1局ごとに手番を入れ替える</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_18">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonSaveSettingsOnly">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>590</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>設定のみ保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonResetToDefault">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>590</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>初期設定に戻す</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
